### PR TITLE
Update sandman_zh_CN.ts

### PR DESCRIPTION
**Pull request by Rain-Clear**
This pull request makes some changes to the existing sandbox with my own thoughts. So I'll keep the pull request in a draft state until I've gathered enough suggestions

Thank you for your contribution to the Sandboxie repository.

In order to reduce the risk of accidental merges, it's highly recommended for beginners to open a new Pull Request in draft state by clicking the button "Create Draft Pull Request", instead of using the default "Create Pull Request" option. 

In addition, you can convert an existing pull request to a draft by clicking the "Convert to draft" link in the right sidebar under "Reviewers".

All new translators are encouraged to look at the "Localization notes and tips" before creating a pull request: https://git.io/J9G19

### CODE REVIEW

**Comment by RimacC2-EV** [comment]
Can someone just take a look at this update or give me some suggestions?
@nkh0472 @love-code-yeyixiao Could you guys take a moment to read it? (If you're busy right now, you can ignore this notification)

File: SandboxiePlus/SandMan/sandman_zh_CN.ts [hunk 1]
```diff
@@ -453,7 +453,7 @@ To disable this template for a sandbox, simply uncheck it in the template list.<
     <message>
         <location filename="Windows/BoxImageWindow.cpp" line="37"/>
         <source>Creating new box image, please enter a secure password, and choose a disk image size.</source>
-        <translation>正在创建新的沙盒磁盘映像，请输入强密码，并设置映像大小。</translation>
+        <translation>正在创建新的沙盒磁盘映像。请输入强密码（可以是字母+数字+符号），并设置映像大小。</translation>
```
Comment (nkh0472) [inline SandboxiePlus/SandMan/sandman_zh_CN.ts:456]: For “secure password”, stick to the original string for translation or adjust it in the upstream source code.

**Review by nkh0472 — COMMENTED** [review]
All good except for one string that recommends adjustment.

File: SandboxiePlus/SandMan/sandman_zh_CN.ts (hunk 1 above)
Comment (RimacC2-EV) [reply SandboxiePlus/SandMan/sandman_zh_CN.ts:456]: Sorry for the late answer······ That's good,and I will change it. Thanks for your advice

**Comment by RimacC2-EV** [comment]
The recommended adjustment has been applied.Could you please review it?

**Review by nkh0472 — COMMENTED** [review]
all good for me

**Pull request merged**